Project.toml

```diff
@@ -10,8 +10,8 @@ Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 UnitfulAstro = "6112ee07-acf9-5e0f-b108-d242c714bf9f"
 
 [compat]
-UnitfulAstro = "0.2.0"
-julia = "1.1.0"
+UnitfulAstro = "^0.2.0"
```

Comment:
`"0.2.0"` and `"^0.2.0"` are equivalent: https://julialang.github.io/Pkg.jl/stable/compatibility/#Version-specifier-format-1

Also, `^0.2.0` evaluates to `[0.2.0, 0.3.0)`, is this intended or rather `[0.2.0, 1.0.0)`, which would be `"0.2.0, 1.0.0"`

Reply:
Right, versions 0.x are truly annoying.